src/foam/dao/history/HistoryRecord.js

```diff
@@ -37,9 +37,9 @@ foam.CLASS({
       name: 'subject',
       documentation: `References subject that made update. TODO: remove references when subject becomes serializable`,
       postSet: function(o, n) {
-        this.userId = n.user.id;
+        this.userId = n.user.id; // this line?
         this.agentId = n.realUser.id;
-        this.user = n.user.toSummary();
+        this.subject.user = n.user.toSummary();
```

Comment:
i think you shouldn't change this line. This updates the local(instance class) variable